client/src/common/components/CardItem.jsx

```diff
@@ -27,7 +28,10 @@ const CardItem = ({
     onClick={onCardClick}
     role="figure"
   >
-    <h3 className="card-item__heading">{name}</h3>
+    <header className="card-item__header">
```

Comment:
Proposal: there is not enough space there, especially in tiles view, especially when the name will be long, maybe move lock icon to some bottom corner?

Reply:
Client wanted to have it next to title